deploy/skeletons/example.py

```diff
@@ -1,8 +1,8 @@
-#-*- coding: utf-8 -*-
-from viur.core.skeleton import Skeleton
 from viur.core.bones import *
+from viur.core.skeleton import Skeleton
+
 
-class exampleSkel(Skeleton):
+class ExampleSkel(Skeleton):
```

Comment:
OK in most cases in our projects, `exampleSkel` is standard. This is ok for me, but then we should update all our projects.

Reply:
We should discuss this in Mousebrandround next monday. Are you available?

Reply:
I think that's a de facto standard naming convention. Even in PEP 8. I think we should follow it.

But, we can talk about it...

Reply:
Yeah, and standards are absolutely welcome. We should just talk about it.